docs/DeveloperGuide.md

```diff
@@ -4,26 +4,167 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Design
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Application Launch
 
```

Comment:
Consider adding a content page or navigation bar with hyperlink to improve navigability and document formatting